src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+                    CheckList c = new CheckList("T", s, false);
+                    inputList.add(c);
+                    System.out.println(dash);
+                    System.out.println("Pai Kia Bot: ok i added this task chop chop: " + c.toString().substring(1));
+                    System.out.println(dash);
+                } else {
+                    System.out.println(dash);
+                    System.out.println("Pai Kia Bot: no description leh, try again");
+                    System.out.println(dash);
+                }
+            } else if (input.length() > 5 && input.substring(0, 6).equals("event ")) {
+                if (!input.equals("event ")) {
+                    String[] s = input.substring(6).split("/at ");
+                    CheckList c = new CheckList("E", s[0] + "(at: " + s[1] + ")", false);
+                    inputList.add(c);
+                    System.out.println(dash);
+                    System.out.println("Pai Kia Bot: ok i added this task chop chop: " + c.toString().substring(1));
+                    System.out.println(dash);
+                } else {
+                    System.out.println(dash);
+                    System.out.println("Pai Kia Bot: no description leh, try again");
+                    System.out.println(dash);
+                }
+            } else if (input.length() > 8 && input.substring(0, 9).equals("deadline ")) {
+                if (!input.equals("deadline ")) {
+                    String[] s = input.substring(9).split("/by ");
+                    CheckList c = new CheckList("D", s[0] + "(by: " + s[1] + ")", false);
+                    inputList.add(c);
+                    System.out.println(dash);
+                    System.out.println("Pai Kia Bot: ok i added this task chop chop: " + c.toString().substring(1));
+                    System.out.println(dash);
+                } else {
+                    System.out.println(dash);
+                    System.out.println("Pai Kia Bot: no description leh, try again");
+                    System.out.println(dash);
+                }
+            } else {
+                System.out.println(dash);
+                System.out.println("You: " + input);
+                if (input.equals("todo") || input.equals("event") || input.equals("deadline")) {
+                    System.out.println("Pai Kia Bot: no description leh, try again");
+                } else {
+                    System.out.println("Pai Kia Bot: invalid response leh");
+                }
+
+                System.out.println(dash);
+            }
+
+            System.out.print("Your response: ");
+            input = sc.nextLine();
+            System.out.println("");
+
+        }
+
+        System.out.println(dash);
+        System.out.println("You: " + input);
+        System.out.println(farewell);
+        System.out.println(dash);
+        sc.close();
     }
 }
+
+class CheckList {
+    String type;
+    String content;
+    boolean checked;
```

Comment:
Perhaps a less ambiguous variable name? If checked means done, then perhaps naming it isDone would be clearer